### PR TITLE
Upgrade from Bootstrap 1.x to 3.x

### PR DESCRIPTION
**Pull request by mistydemeo**
- Remove unused vendored Bootstrap and jQuery.
- Update class names.
- Use the Django Bootstrap helper we also use in artefactual/archivematica#374.

### CODE REVIEW

File: fpr/templates/format/form.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% extends "fpr/app_layout.html" %}
 {% load url from future %}
+{% load bootstrap_tags %}
```
Comment (Hwesta) [inline fpr/templates/format/form.html:2]: What dependency provides this?

Comment (mistydemeo) [inline fpr/templates/format/form.html:2]: django-forms-bootstrap, which is provided by the hosting app (Archivematica or the fpr-server).

**Comment by Hwesta** [comment]
Despite the name, I think the branch of the fpr-admin this should be merged against is 1.x.  a022ffb1d905a1889a7acea53b59bc0ba291156e contains the squashed commit of https://github.com/artefactual/archivematica-fpr-admin/compare/1.x...master?expand=1 Can you make a new PR against 1.x?

**Comment by mistydemeo** [comment]
Replaced by #17.

**Pull request closed without merging**